runtime/near-vm-runner/src/wasmtime_runner/logic.rs

```diff
@@ -3925,22 +3933,25 @@ fn read_and_parse_account_id(
     registers: &Registers,
     ptr: u64,
     len: u64,
-) -> Result<AccountId> {
+    config: &Config,
```

Comment:
context argument are usually passed first, so please move `config` to be after `registers`